src/calculateRentalCost.js

```diff
@@ -3,8 +3,23 @@
  *
  * @return {number}
  */
+/**
+ * Calculates the rental cost based on the number of days rented,
+ * applying different discounts based on the duration.
+ *
+ * @param {number} days - The number of days the item is rented for.
+ * @return {number} The total rental cost.
+ */
 function calculateRentalCost(days) {
-  // write code here
+  const dailyRate = 40;
+
+  if (days < 3) {
+    return days * dailyRate;
+  } else if (days >= 3 && days <= 6) {
+    return days * dailyRate - 20;
+  } else {
+    return days * dailyRate - 50;
+  }
```

Comment:
Avoid using hard-coded literals for the discount values and the threshold number of days. Define them as constants at the beginning of your function to make the code more readable and maintainable.

Reply:
Please follow this comment, create variables for all numbers here 
https://refactoring.guru/replace-magic-number-with-symbolic-constant